__init__.py

```diff
@@ -554,7 +555,7 @@ def handle_forecast(self, message):
         # Get a date from spoken request
         when = self.__extract_datetime(message.data.get('utterance'),
                                 lang=self.lang)[0]
-        today = self.__extract_datetime("today")[0]
+        today = now_local()
```

Comment:
should hour, minute and second be set to 0 (or converted to Date) to match old behaviour?

Reply:
Good call.